pyrdp/mitm/crawler_config/match.txt

```diff
@@ -0,0 +1,25 @@
+# Common extensions
+*/*.txt
+*/*.pdf
+*/*.csv
+*/*.xls*
+*/*.doc*
+*/*.ppt*
+
+# Possible malware samples
+*/*.bat
+*/*.exe
+*/*.dll
+*/*.ps1
```

Comment:
.py? possibly other languages as well

Reply:
Do you have any input on this? @obilodeau This file is what we currently download from the disk, folder AND files.

Reply:
I think that crawling code files would give us a lot of thrash and would expand the crawl time. I went with compiled files and scripts that work out-of-the-box on Windows / Linux.